web/crux/src/app/deploy/interceptors/deploy.start.interceptor.ts

```diff
@@ -124,6 +129,34 @@ export default class DeployStartValidationInterceptor implements NestInterceptor
       }
     }
 
+    if (deployment.configBundles.length > 0) {
+      const deploymentEnv = (deployment.environment as UniqueKeyValue[]) ?? []
+      const deploymentEnvKeys = deploymentEnv.map(it => it.key)
+
+      const seenKeys: Record<string, string> = {} // Environment key -> config bundle name
```

Comment:
Do not really understand the comment in here. Could you please elaborate a bit more? I can't came up with a better naming but `seenKeys` a bit shady for me 🍨

Reply:
What about checkedKeys?

Reply:
Better for me

Reply:
With the comment I wanted to suggest that it's a mapping which connects environment keys to config bundle names. Maybe using `envToBundle` would be a better variable name, and I could then remove the comment?

Reply:
@robot9706 good idea, yea.